queries.yaml

```diff
@@ -160,6 +160,7 @@ post_process:
         id: true
         source: true
         boundary: "yes"
+      snap_tolerance: 0.125
```

Comment:
Should be okay across all zooms, since neither the calculation nor the tolerance depends on the zoom.